app/models/vm_or_template.rb

```diff
@@ -1193,7 +1202,7 @@ def self.assign_ems_created_on_queue(vm_ids)
     MiqQueue.put(
       :class_name  => name,
       :method_name => 'assign_ems_created_on',
-      :args        => [vm_ids],
+      :args        => Array(vm_ids),
```

Comment:
I think this is supposed to be a nested Array, e.g. `[Array(vm_ids)]`  It's some quirk with the queue args i can't quite remember.